pip/compat.py

```diff
@@ -41,7 +46,14 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                try:
+                    from ctypes import cdll
+                    return s.decode("cp" + str(cdll.kernel32.GetACP()))
```

Comment:
backslashreplace is usable for decode in Python 3.5+ (from the docs) hence my comment about only using it there. ```replace``` is OK, though - it just loses a bit more information, but that's OK.